Examples/Python/src/Traccc.cpp

```diff
@@ -35,45 +35,44 @@ using namespace ActsExamples;
 namespace Acts::Python {
 
 void addTraccc(Context& ctx) {
-  auto [m, mex] = ctx.get("main", "examples");
-
-  auto traccc = mex.def_submodule("traccc");
+  auto [_, mex] = ctx.get("main", "examples");
+  auto m = mex.def_submodule("_traccc");
```

Comment:
Does this run the code that adds a submodule multiple times?

Reply:
We anyway need to streamline a bit the python bindings and make it somewhat consistent.